keyboards/alt34/rules.mk

```diff
@@ -0,0 +1,21 @@
+# MCU name
```

Comment:
Also, the content could be moved to the rev1 folder, and would remove the need some of the includes you have.

Reply:
You mean that the entire contents of my `alt34/rules.mk` could be moved to the `alt34/rev1/rules.mk`?

Reply:
correct.  And no, actually, youd want to remove .t